src/modules/mc_hover_thrust_estimator/MulticopterHoverThrustEstimator.cpp

```diff
@@ -149,28 +167,30 @@ void MulticopterHoverThrustEstimator::Run()
 			reset();
 		}
 
+		ZeroOrderHoverThrustEkf::status status;
 		status.hover_thrust = _hover_thrust_ekf.getHoverThrustEstimate();
 		status.hover_thrust_var = _hover_thrust_ekf.getHoverThrustEstimateVar();
 		status.accel_noise_var = _hover_thrust_ekf.getAccelNoiseVar();
 		status.innov = NAN;
 		status.innov_var = NAN;
 		status.innov_test_ratio = NAN;
-	}
 
-	publishStatus(status);
+		publishStatus(status);
+	}
 
 	perf_end(_cycle_perf);
 }
 
 void MulticopterHoverThrustEstimator::publishStatus(ZeroOrderHoverThrustEkf::status &status)
 {
-	hover_thrust_estimate_s status_msg{};
+	hover_thrust_estimate_s status_msg;
 	status_msg.hover_thrust = status.hover_thrust;
 	status_msg.hover_thrust_var = status.hover_thrust_var;
 	status_msg.accel_innov = status.innov;
 	status_msg.accel_innov_var = status.innov_var;
 	status_msg.accel_innov_test_ratio = status.innov_test_ratio;
 	status_msg.accel_noise_var = status.accel_noise_var;
+	status_msg.valid = (status.hover_thrust_var < 0.01f) && (status.innov_test_ratio < 1.f);
```

Comment:
TODO: review

Reply:
With the hysteresis, I think the numbers make sense.